Tasks/Kubernetes/src/kubernetes.ts

```diff
@@ -74,4 +84,16 @@ function executeKubectlCommand(clusterConnection: ClusterConnection) : any {
         tl.setResult(tl.TaskResult.Failed, err.message);
     })
     .done();
+}
+
+// execute kubectl create configmap command
+function executeCreateConfigMapCommand(clusterConnection: ClusterConnection, configMapName: string) : any {
+   
+    kubectlConfigMap.run(clusterConnection, configMapName).fin(function cleanup(){
+        clusterConnection.close();
+    }).then(function success() {
+        executeKubectlCommand(clusterConnection);
```

Comment:
the function name is executeCreateConfigMapCommand, it should not run any other kubectl command. You need to refactor the code.